app/server/routes/api.ts

```diff
@@ -227,6 +231,12 @@ router.get("/known-issues", async (_, response) => {
 
 router.post("/contact-us", contactUsFormHandler);
 
-router.post("/reminder", reminderHandler);
+router.post("/reminders", createReminderHandler);
+router.get(
+  "/reminders/status",
+  membersDataApiHandler("user-attributes/me/reminders", "MDA_REMINDERS_STATUS")
+);
+router.post("/reminders/cancel", cancelReminderHandler);
+router.post("/reminders/reactivate", reactivateReminderHandler);
```

Comment:
might be worth factoring this out into its own router? and then

```js
router.use("/reminders", remindersRouter);
```

Reply:
In this case there is no need. We already use this pattern in the file. All the api endpoints are declared here but their logic is elsewhere. That way we don't have to go several levels deep to know which endpoints we are using.